css/index.css

```diff
@@ -72,8 +72,8 @@ h1, h2, h3, h4, h5 {
 }
 
 body{
-	margin-left: 10px;
-	margin-right: 10px;
+	margin-left: 1.14%;
```

Comment:
If you're thinking of refactoring this, it can be condensed to be `margin: 0 1.14%`